terminal_api_flows/flows/credit_txn.py

```diff
@@ -5,10 +5,85 @@
 from terminal_api_flows.tools.decorators import terminal_ping_decorator_3_attempts
 
 
+CARD_DECLINE_STATUSES = [
+    "NFC_READ_ERROR",
+    "DIP_READ_ERROR",
+    "SWIPE_READ_ERROR",
+]
+
+
 # *********************** #
 # Credit Transaction Flow #
 # *********************** #
 
+@terminal_ping_decorator_3_attempts
+def credit_to_cash_transaction(total=0):
+    json_data, discount = get_customers()
+
+    pos_checkout_id, pos_order_id, customer_uid = generate_ids(json_data)
+
+    checkout_data = {
+        "checkout": {
+            "pos_checkout_id": pos_checkout_id,
+            "type": "CREDIT",
+            "total": total,
+            "customer_account_uid": customer_uid,
+            "discounts_applied": discount
+        },
+        "order": {
+            "currency": "USD",
+            "pos_order_id": pos_order_id,
+            "products": [
+                {
+                    "name": "hamburger",
+                    "price": 250,
+                    "price_with_vat": 0,
+                    "quantity": 2,
+                    "receipt_nest_level": 1,
+                    "single_vat_amount": 0,
+                    "total_price": 500,
+                    "total_with_vat": 0,
+                    "vat_rate": 0,
+                    "vat_amount": 0
+                }
+            ],
+            "subtotal": 0,
+            "tax": 125
+        }
+    }
+
+    # POST to checkouts
+    res, json_data = http_request("checkouts", "POST", json.dumps(checkout_data).encode("utf-8"))
+
+    if res.status == 200 and json_data["status"] == "TRANSACTION_STARTED":
+        res, json_data = http_request(f"checkouts/{pos_checkout_id}", "GET")
+
+        status = json_data["status"]
+
+        if status == "SUCCESSFUL":
+            print_outcome("SUCCESS", res.status, json_data)
+            exit(0)
+
+        print(f"Transaction Status: {status}")
+
+        if status in CARD_DECLINE_STATUSES:
```

Comment:
I think adding this will help:
```
print(f"Credit Card Status: {status}")
```
That's how I found out there is a decline error message coming that is not covered by these: 
```
CARD_DECLINE_STATUSES = [
    "NFC_READ_ERROR",
    "DIP_READ_ERROR",
    "SWIPE_READ_ERROR",
]
```